sentences/es/fan_HassTurnOff.yaml

```diff
@@ -3,7 +3,7 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "apaga [todos] [los] ventiladores <area>"
+          - "<apaga> [todos] [los] ventiladores [(de|en)] <area>"
```

Comment:
the `[(de|en)]` is not needed as it's already included in the `<area>` expansion rule

Reply:
Fixed